x-pack/plugins/cases/server/saved_object_types/migrations/user_actions/index.test.ts

```diff
@@ -11,34 +11,51 @@ import {
   SavedObjectMigrationContext,
```

Comment:
nit: to clean up eventually we might want to split this up into a file for the tests for `connector_id.ts` and the new payload migrations.